tools/scripts/generate-sponsors.mts

```diff
@@ -1,202 +1,100 @@
 import fetch from 'cross-fetch';
 import * as fs from 'fs';
 import * as path from 'path';
-import prettier from 'prettier';
 
 import { PACKAGES_WEBSITE } from './paths.mts';
 
-const graphqlEndpoint = 'https://api.opencollective.com/graphql/v2';
-
-const queries = {
-  account: `{
-    account(slug: "typescript-eslint") {
-      orders(status: ACTIVE, limit: 1000) {
-        totalCount
-        nodes {
-          tier {
-            slug
-          }
-          fromAccount {
-            id
-            imageUrl
-            name
-            website
-          }
-        }
-      }
-    }
-  }`,
-  collective: `{
-    collective(slug: "typescript-eslint") {
-      members(limit: 1000, role: BACKER) {
-        nodes {
-          account {
-            id
-            imageUrl
-            name
-            website
-          }
-          tier {
-            amount {
-              valueInCents
-            }
-            orders(limit: 100) {
-              nodes {
-                amount {
-                  valueInCents
-                }
-              }
-            }
-          }
-          totalDonations {
-            valueInCents
-          }
-          updatedAt
-        }
-      }
-    }
-  }`,
-};
-
-interface AccountData {
-  orders: {
-    nodes: OrderNode[];
-  };
-}
-
-interface OrderNode {
-  fromAccount: MemberAccount;
-  tier?: Tier;
-}
-
-interface Tier {
-  slug: string;
-}
-
-interface CollectiveData {
-  members: {
-    nodes: MemberNode[];
+type MemberNodes = {
+  account: {
+    id: string;
+    imageUrl: string;
+    name: string;
+    website: string;
   };
-}
-
-interface MemberNode {
-  account: MemberAccount;
-  totalDonations: {
-    valueInCents: number;
-  };
-}
-
-interface MemberAccount {
-  id: string;
-  imageUrl: string;
-  name: string;
-  website: string;
-}
+  totalDonations: { valueInCents: number };
+}[];
 
 const excludedNames = new Set([
-  'Guest', // Apparent anonymous donor equivalent without an avatar
   'Josh Goldberg', // Team member 💖
 ]);
 
 const filteredTerms = ['casino', 'deepnude', 'tiktok'];
 
-async function requestGraphql<Data>(key: keyof typeof queries): Promise<Data> {
-  const response = await fetch(graphqlEndpoint, {
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({ query: queries[key] }),
-  });
-
-  const { data } = (await response.json()) as {
-    data: Record<typeof key, unknown>;
-  };
-  return data[key] as Data;
-}
-
-async function main(): Promise<void> {
-  const [account, collective] = await Promise.all([
-    requestGraphql<AccountData>('account'),
```

Comment:
all this data was not adding anything, so I removed it, so we're now down to a single GraphQL query 🙂